packages/palette/src/elements/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -370,9 +373,13 @@ const AutocompleteInputDropdown = styled(Box)`
   z-index: 1;
 `
 
-const AutocompleteInputOptions = styled(Box)`
+interface AutocompleteInputOptionProps {
+  fullHeight?: boolean
+}
+
+const AutocompleteInputOptions = styled(Box)<AutocompleteInputOptionProps>`
   /* 308 = Roughly, 5.5 default sized options  */
-  max-height: 308px;
+  max-height: ${(props) => (props.fullHeight === true ? "none" : "308px")};
```

Comment:
Wouldn't a better interface be simply accepting a `maxHeight` option?

Reply:
I like your idea. I called the options `dropdownMaxHeight` though, to avoid confusion